src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -23,13 +23,32 @@
     private static final double WINDOW_WIDTH = 1000;
     private static final double WINDOW_HEIGHT = 500;
 
+    // login page vars
     private Stage thestage;                         // Holds the scene that user is currently viewing
     private Scene signInScene, mainScene;           // Scenes to hold all the elements for each page
     private Button signInButton;
     private TextField userInput;
     private PasswordField passInput;                // Takes input for username and password
     private ClientContext clientContext;            //
 
+    // main page vars
+    private HBox hboxClient;
```

Comment:
Do all of these need to be class-level variables? Can some of them be local variables? For variables that do need to be class-level variables, you should provide comments explaining what they're used for.